cobalt/renderer/rasterizer/skia/text_shaper.cc

```diff
@@ -426,9 +428,9 @@ void TextShaper::ShapeSimpleRun(const char16_t* text_buffer, size_t text_length,
   int glyph_count = 0;
   Font* last_font = NULL;
 
+  base::StringPiece16 text(text_buffer, text_length);
```

Comment:
What does adding base::StringPiece16 change in this file?

Reply:
It bounds text_length to text_buffer so that the iterator does not visit out-of-bound indexes of the buffer.